src/video/canvas.cpp

```diff
@@ -63,6 +63,8 @@ Canvas::render(Renderer& renderer, Filter filter)
 
   Painter& painter = renderer.get_painter();
 
+  //auto viewport_backup = m_context.get_viewport();
```

Comment:
What is this needed for?

Reply:
Leftovers of experimentations, will remove